core/src/components/cat-tabs/cat-tabs.tsx

```diff
@@ -99,44 +122,174 @@ export class CatTabs {
     this.hostElement.tabIndex = Number(this.hostElement.getAttribute('tabindex')) || 0;
     return (
       <Host>
-        {this.tabs.map((tab: HTMLCatTabElement) => {
-          return (
-            <cat-button
-              buttonId={tab.id}
-              role="tab"
-              part="tab"
-              class={{
-                'cat-tab': true,
-                'cat-tab-active': tab.id === this.activeTab,
-                'cat-tab-error': tab.error
-              }}
-              active={tab.id === this.activeTab}
-              color={tab.error ? 'danger' : tab.id === this.activeTab ? 'primary' : 'secondary'}
-              variant="text"
-              icon={tab.icon ? (tab.error ? '$cat:input-error' : tab.icon) : undefined}
-              iconOnly={tab.iconOnly}
-              iconRight={tab.iconRight}
-              url={tab.url}
-              disabled={tab.deactivated}
-              urlTarget={tab.urlTarget}
-              onCatClick={() => this.click(tab)}
-              testId={tab.testId}
-              nativeAttributes={{ ...tab.nativeAttributes }}
-              nativeContentAttributes={{ 'data-text': tab.label }}
-              data-dropdown-no-close
-            >
-              {tab.label}
-            </cat-button>
-          );
-        })}
-        <slot name="more"></slot>
```

Comment:
removing the named slot is a breaking change. Its not a problem but we need to remember it when building the new version and updating. I guess your team is the only one using it right now